### PR TITLE
Fix translations in credits.

### PR DESCRIPTION
**Pull request by juanfra**
Currently, we're not translating the URL to WordPress but the doc for translators says that.

We should be making the URL to WordPress translatable, so each locale can have their own link. And at the same time being consistent with the doc for translators. Not sure if the usage of `_x()` is correct there.

**Fixes:** #301 
**WP Username:** juanfra

### CODE REVIEW

File: footer.php [hunk 1]
```diff
@@ -131,10 +131,12 @@
...
 
 						<p class="powered-by-wordpress">
-							<?php
-							/* Translators: %s = Link to WordPress.org */
-							printf( esc_html_x( 'Powered by %s', 'Translators: %s = Link to WordPress.org', 'twentytwenty' ), '<a href="https://wordpress.org">' . esc_html__( 'WordPress', 'twentytwenty' ) . '</a>' );
-							?>
+							<a href="<?php echo esc_url( __( 'https://wordpress.org/', 'twentytwenty' ) ); ?>">
+								<?php
+								/* Translators: %s: WordPress. */
+								printf( __( 'Powered by %s.', 'twentytwenty' ), __( 'WordPress', 'twentytwenty' );
```
Comment (pattonwebz) [inline footer.php]: Missing closing bracket here

Comment (juanfra) [reply footer.php]: Oh sorry for that!

**Comment by juanfra** [comment]
Perfect, thanks :)

We can either wait for a decision on https://github.com/WordPress/twentytwenty/pull/177 or I can update the code to use the escaped version.

**Comment by pattonwebz** [comment]
In line with the other PR about similar translation escapes I expect that whitelisting the missing escape with `// phpcs:ignore` comment is the best option here.

**Comment by juanfra** [comment]
Thank you @pattonwebz ! Perfect, I updated it.

**Pull request merged**